knimin/handlers/ag_add_barcode_kit.py

```diff
@@ -7,11 +7,13 @@
 class AGAddBarcodeKitHandler(BaseHandler):
```

Comment:
I can add that, although that requires adding a base access level to the database as well.